packages/plugin-core/src/WorkspaceWatcher.ts

```diff
@@ -334,7 +334,6 @@ export class WorkspaceWatcher {
       // eslint-disable-next-line  no-async-promise-executor
       const p = new Promise(async (resolve) => {
         note.updated = now;
-        await engine.updateNote(note);
```

Comment:
Don't we still need this?

Reply:
1. Since vscode fires 2 events for a save event (onWillSave and onDidSave), we only want to update the engine once (otherwise we will also fire 2 engine events). The second event that is handled by TextDocumentService will contain all the info needed to update the engine.
2. This is more of a side effect, but `note.updated = now` actually updates the engine note already

Reply:
Derp - we discussed this already haha.  Got it, makes sense.